scss/layout/_scaffolding.scss

```diff
@@ -0,0 +1,122 @@
+@import '../abstracts/variables';
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+[class*='container'] {
+  margin: 0 auto;
+  padding-right: $column-gutter;
+  padding-left: $column-gutter;
+}
```

Comment:
We definitely should not take this class, it's been super annoying to use so let's not migrate this over.